core-contracts/FeeDistribution/src/main/java/finance/omm/score/fee/distribution/AbstractFeeDistribution.java

```diff
@@ -90,6 +94,10 @@ public void FeeClaimed(Address caller, Address receiver, BigInteger amount) {
     public void FeeDistributed(BigInteger _value) {
     }
 
+    @EventLog(indexed = 1)
```

Comment:
```suggestion
    @EventLog
```
You don't need to index the first argument here, since the _value is pretty arbitrary and cannot be searchable.

Reply:
Other than this, it looks good.

Reply:
Got it